_posts/zj

```diff
@@ -0,0 +1,9 @@
+@@ -0,0 +1,18 @@
+---
+layout: post
+title: Yien
```

Comment:
Please change title to something that is not already used. Your current title is conflicting with https://github.com/sunwaytechclub/hacktoberfest-2019/blob/master/_posts/2019-9-18-Yien.md